src/HtmlGenerator/Program.cs

```diff
@@ -17,201 +17,17 @@ public class Program
-                    continue;
-                }
-
-                if (string.Equals(arg, "/donotincludereferencedprojects", StringComparison.OrdinalIgnoreCase))
-                {
-                    doNotIncludeReferencedProjects = true;
-                    continue;
-                }
-
-                if (arg == "/nobuiltinfederations")
-                {
-                    noBuiltInFederations = true;
-                    Log.Message("Disabling built-in federations.");
-                    continue;
-                }
-
-                if (arg.StartsWith("/federation:"))
-                {
-                    string server = arg.Substring("/federation:".Length);
-                    Log.Message($"Adding federation '{server}'.");
-                    federations.Add(server);
-                    continue;
-                }
-
-                if (arg.StartsWith("/offlinefederation:"))
-                {
-                    var match = Regex.Match(arg, "/offlinefederation:(?<server>[^=]+)=(?<file>.+)");
-                    if (match.Success)
-                    {
-                        var server = match.Groups["server"].Value;
-                        var assemblyListFileName = match.Groups["file"].Value;
-                        offlineFederations[server] = assemblyListFileName;
-                        Log.Message($"Adding federation '{server}' (offline from '{assemblyListFileName}').");
-                        continue;
-                    }
-                    continue;
-                }
-
-                if (string.Equals(arg, "/noplugins", StringComparison.OrdinalIgnoreCase))
-                {
-                    SolutionGenerator.LoadPlugins = false;
-                    continue;
-                }
-
-                if (string.Equals(arg, "/useplugins", StringComparison.OrdinalIgnoreCase))
-                {
-                    SolutionGenerator.LoadPlugins = true;
-                    continue;
-                }
-
-                if (arg.StartsWith("/noplugin:"))
-                {
-                    pluginBlacklist.Add(arg.Substring("/noplugin:".Length));
-                    continue;
-                }
-
-                if (arg == "/excludetests")
-                {
-                    SolutionGenerator.ExcludeTests = true;
-                    continue;
-                }
-
-                if (arg.StartsWith("/root:", StringComparison.Ordinal))
-                {
-                    rootPath = Path.GetFullPath(arg.Substring("/root:".Length).StripQuotes());
-                    continue;
-                }
-
-                try
-                {
-                    AddProject(projects, arg);
-                }
-                catch (Exception ex)
-                {
-                    Log.Write("Exception: " + ex.ToString(), ConsoleColor.Red);
-                }
-            }
-
-            if (projects.Count == 0)
+            if (options.Properties.Count == 0)

```

Comment:
Thought I was typing the other thing starting with `options.Pro`. Fix: https://github.com/KirillOsenkov/SourceBrowser/pull/173